src/services/blockchain/ethereum/ethereum-blockchain-service.ts

```diff
@@ -66,20 +66,15 @@ async function attempt<T>(
  * @param walletBalance - Available funds.
  */
 function handleInsufficientFundsError(txData: TransactionRequest, walletBalance: BigNumber): void {
-  const txCost = (txData.gasLimit as BigNumber).mul(txData.maxFeePerGas)
+  const txCost = (txData.gasLimit as BigNumber).mul(txData.maxFeePerGas || 1)
```

Comment:
Can you put this `1` as a constant somewhere? Out of curiosity, why `1` is the right number :) ?

Reply:
That's to tell TS it's a safe operation. Going to add a comment there.

Reply:
Or, rather change the code.